portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -19,14 +19,28 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
+import java.util.ArrayList;
 
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  String[] rankings = {"VOSS", "FIJI", "SMART WATER", "CRYSTAL GEYSER"};
```

Comment:
I know it's all a practice but you can enhance it a bit. for example, don't put these at the constructor of your servlet. it does not happen in real-life applications. Servlet loads the data from their databases.

Reply:
How would you recommend then holding the rankings?